packages/grafana-data/src/types/plugin.ts

```diff
@@ -111,6 +112,40 @@ export interface PluginDependencies {
   grafanaDependency?: string;
   grafanaVersion: string;
   plugins: PluginDependencyInfo[];
+  extensions: {
+    // A list of exposed component IDs
+    exposedComponents: string[];
+  };
+}
+
+export interface PluginExtensions {
+  // The component extensions that the plugin registers
+  addedComponents: Array<{
+    targets: string[];
+    title: string;
+    description: string;
+  }>;
```

Comment:
nit: would it make sense to introduce a type here? 

```ts
type ExtensionInfo = {
   targets: string[];
   title: string;
   description;
}
```

Reply:
Should be fixed in https://github.com/grafana/grafana/pull/93429/commits/a721c7ca42ea28f36df7b525bdea64bcc1adf9ce 👍